tree/dataframe/src/lexertk.hpp

```diff
@@ -774,7 +774,7 @@ class helper_interface {
 
 class token_scanner : public helper_interface {
 public:
-   virtual ~token_scanner() {}
+   ~token_scanner() override {}
```

Comment:
I would leave `lexertk.hpp` alone, we are just vendoring this third-party project, we are not actually maintaining the code. applying changes like these might make it harder to update it or patch it.

on the other hand, if we are never going to update it or patch it, we might as well treat it as our own code. up to @dpiparo and @vepadulano I guess.

Reply:
You are right! It has been also our policy in similar situations to not change the file, so I will revert the changes to this file in this PR